### PR TITLE
html, css, javascript linter configuration

### PR DESCRIPTION
**Pull request by kennedyowusu**
### Leaderboard Project

- [ ] Create the layout of the project
- [ ] Apply minimal style
- [ ] Use webpack to bundle the project
- [ ] Use proper ES6 syntax
- [ ] Ensure all linters pass

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,611 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack serve --open"
```
Comment (zieeco) [inline package.json]: - Hey, your package.json is Missing the script: "start", kindly add this `"start": "webpack serve --open",` to be able to run your app.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        <h2 class="main-left-title">Recent scores</h2>
+        <button class="refresh">Refresh</button>
+       </div>
+       <ul class="main-left-list" id="mainLeftList">
+         <!-- list of recent scores -->
+
+         <!-- <li class="main-left-list-items">
+           <span class="name" id="userName">Name: </span>
+           <span class="score" id="userScore">100 </span>
+         </li> -->
```
Comment (zieeco) [inline src/index.html]: - Let's keep our codebase clean by removing unused commented code.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  computeScores();
+  generateAllScores();
+};
+
+// Event listeners
+submitScore.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  validateFields();
+});
```
Comment (zieeco) [inline src/index.js]: - Hey, you're having some console error, please, check to fix the issue with your app not showing the board except for the page title.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @kennedyowusu ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice github flow. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (kennedyowusu) [reply package.json]: Resolved this issue as seen in the image attached below

add-start

File: src/index.html (hunk 2 above)
Comment (kennedyowusu) [reply src/index.html]: Comments removed from code base as can be seen in the image below. 

remove-commented-code

File: src/index.js (hunk 3 above)
Comment (kennedyowusu) [reply src/index.js]: I have Resolve this also. I have attached an error free console image below.


error-free-console

**Comment by kennedyowusu** [comment]
I have resolved all suggestions raised by the reviewer and I have attached images to prove that the issues have been resolved.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @kennedyowusu, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**